Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -34,14 +34,24 @@
     - GlassesCorpsman # Added for prescription glasses.
     - HamsterWearable
     - WhitelistChameleon
-    - SecDogWearable 
+    - SecDogWearable
   - type: GuideHelp
     guides:
     - Security
     - Antagonists
     - Medical
   - type: IdentityBlocker
     coverage: EYES
+  - type: MedSecHud # we have ComponentToggler at home
```

Comment:
why does this not just use ComponentToggler

Reply:
because we need to both add some and remove some components on activation 
componentToggler would just add them on activation and then remove the same set of components when it's deactivated

Reply:
add the inverse to it and make upstream pr too

Reply:
yeah real